File: server/makc2022--nodejs--nestjs/src/layer1/query/query-helper.ts
/** Copyright (c) 2022 Maxim Kuzmin. All rights reserved. Licensed under the MIT License. */

import { Guid } from 'guid-typescript';

/** Помощник запроса. */
export class QueryHelper {
  /** Создать код запроса.
   * @returns Код запроса.
   */
  static createQueryCode(): string {
    return Guid.create().toString().replace(/-/g, '').toUpperCase();
  }
}

File: server/makc2022--dotnet/Makc2022.Layer1/Query/QueryHelper.cs
// Copyright (c) 2022 Maxim Kuzmin. All rights reserved. Licensed under the MIT License.

namespace Makc2022.Layer1.Query
{
    /// <summary>
    /// Помощник запроса.
    /// </summary>
    public class QueryHelper
    {
        #region Public methods

        /// <summary>
        /// Создать код запроса.
        /// </summary>
        /// <returns>Код запроса.</returns>
        public static string CreateQueryCode()
        {
            return Guid.NewGuid().ToString("N").ToUpper();
        }

        #endregion Public methods
    }
}
